.github/copilot-instructions.md

```
@@ -224,9 +224,44 @@ protected function isAccessible(User $user, ?string $path = null): bool
+   - @tailwind components;
+   - @tailwind utilities;
+   + @import "tailwindcss";
+</code-snippet>
+
+### Replaced Utilities
+- Tailwind v4 removed deprecated utilities. Do not use the deprecated option; use the replacement.
+- Opacity values are still numeric.
+
+| Deprecated |	Replacement |
+|------------+--------------|
+| bg-opacity-* | bg-black/* |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)